classes/03class/exercises/c03-docker01/feernandobraga/README.md

```diff
@@ -0,0 +1,42 @@
+# C03-Docker01
+
+## Docker
+
+- [Dockerfile](Dockerfile)
+
+## Command Execution Output
+
+- Command to build the image:
+
+  ```
+  docker build -t da-terraform:c03-docker-e01 .
+  ```
+
+- Command to run the image and its output.:
+
+  - listing the images
+
+    ```
+    docker image ls
+
+    response:
+    REPOSITORY          TAG                 IMAGE ID            CREATED             SIZE
+    da-terraform        c03-docker-e01      317884cb2010        2 minutes ago       56MB
+
+    ```
+
+  - running the image
+
+    ```
+    docker run 3178
```

Comment:
this command doesn't seem to be the right one..

Reply:
Hi Kiko,
I can run images and containers by referring to the first 4 digits of its id.
Therefore, based on my `docker image ls` command, the ID for the image da-terraform is 317884cb2010.
Hence, I can run the image by typing `docker run 3178`.
Please let me know if you still want me to change it.

Reply:
hi Fernando..
I never used that, so nice to know..